stylus/components/nav.styl

```diff
@@ -68,13 +68,11 @@
     &:visited
         color  slate-grey
 
-    .active // deprecated
-    .is-active
     &:hover
         color  charcoal-grey
 
-    .active // deprecated
-    .is-active
+    &.active // deprecated
+    &.is-active
```

Comment:
Did you experience any issue to put those `&` back? I removed them on purpose to reduce the weight of selectors.

Reply:
OK for the fix. For the record, we should use BEM here.

Reply:
@GoOz Yep, there wasn't any `active` styles anymore on my side

Reply:
Oh ok. I didn't get this bug on Drive but i suppose it's done differently on your app, that's why. Ok to re-add it then. For now at least. :)

Reply:
Drive seems to use Link component differently indeed, keep me noticed if you have some issues about this point so.